inference-engine/samples/common/utils/src/args_helper.cpp

```diff
@@ -326,3 +359,146 @@ void printInputAndOutputsInfo(const InferenceEngine::CNNNetwork& network) {
                   << layer.second->getLayout() << std::endl;
     }
 }
+void configurePrePostProcessing(std::shared_ptr<ov::Function>& function,
+                                const std::string& ip,
+                                const std::string& op,
+                                const std::string& iop,
+                                const std::string& il,
+                                const std::string& ol,
+                                const std::string& iol,
+                                const std::string& iml,
+                                const std::string& oml,
+                                const std::string& ioml) {
+    auto preprocessor = ov::preprocess::PrePostProcessor();
+    std::unordered_map<size_t, ov::preprocess::InputInfo> inputs_info;
+    std::unordered_map<size_t, ov::preprocess::OutputInfo> outputs_info;
+    const auto inputs = function->inputs();
+    const auto outputs = function->outputs();
+    if (!ip.empty()) {
+        auto type = getType(ip);
+        for (size_t i = 0; i < inputs.size(); i++) {
+            get_info(inputs_info, i).tensor(ov::preprocess::InputTensorInfo().set_element_type(type));
```

Comment:
getters for preprocessing are implemented, I suppose we can continue with this PR

Reply:
PR was updated